src/posts/2021-02-15-phaser-blog.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Phaser'
+comments: true
+author: eline
+date: 2020-02-15
+---
+
+### Wat is Phaser?
+Kort gezegd is [Phaser](https://phaser.io/) een gameframework waarmee je HTML-5 games kan maken. Deze games kunnen gespeeld worden op desktop en mobiel. Phaser kan gratis gedownload worden via https://phaser.io/ en is ontwikkeld door Photon Storm in 2013.
+
+Om gebruik te maken van Phaser is het handig als je al enige kennis hebt van HTML en JavaScript. Phaser is namelijk een JavaScript bibliotheek, wat dus betekent dat je de game volledig hierin bouwt. Phaser maakt het mogelijk voor beginners en gevorderden om een eigen interactieve en leuke game te maken. Verder heeft het veel verschillende functies, zelf te veel om allemaal te benoemen. Dit is ook een van de redenen waarom het zo geliefd is bij game ontwikkelaars. 
+
+### Opzetten van een Phaser game
+Het begint allemaal bij je HTML bestand waar je Phaser oproept in je head, met de volgende code: 
+```
+<script src="https://cdnjs.cloudflare.com/ajax/libs/phaser-ce/2.12.0/phaser.min.js"></script>
+```
+Na dit gedaan te hebben maak je een JavaScript bestand aan en roep je de game op met:
+```
+window.onload = function(){
+    var game = new Phaser.Game(800 (width), 600 (height), Phaser.AUTO, "", {
+        preload: preload,
+        create: create,
+        update: update
+  }); 
+```
+Als weergavecontext voor de game kan er gekozen worden voor Phaser.CANVAS, Phaser.WEBGL of Phaser.AUTO. De aanbevolen waar om te gebruiken is Phaser.AUTO, omdat deze automatisch dat gebruikt wat de browser die wordt gebruikt ondersteunt. De width en de height staan voor de canvasgrootte, in dit geval is dat 800 x 600 pixels. 
+
+#### Preload functie
+Verder heeft elke game een preload, create en update functie, waarmee je het spel maakt. Wanneer de game start zoekt Phaser automatisch naar de preload functie en laadt alles wat daar in staat. In de preload roep je alle ‘’game.load’’ aan om een wachtrij voor de lader op te bouwen. 
+
+#### Create functie
+Zodra het laden van de preload functie klaar is wordt de create functie automatisch gestart. In de create functie staat het grootste deel van de set-upcode. Het is belangrijk om te weten dat de game objecten worden weergegeven in de volgorde waarin ze worden aangemaakt. Je kan zelf kiezen waar je objecten op het canvas wil plaatsen. Dit doe je door ook de variabelen width en height aan ze mee te geven. Het is zelfs mogelijk om het object buiten de 0, 0 tot 800, 600 te plaatsen. Je kan het object dan niet zien maar het zal nog steeds binnen de scene aanwezig zijn. Met het camerasysteem van Phaser 3 is het mogelijk om met een actieve camera je in het spel te verplaatsen en in en uit te zoomen. 
```

Comment:
_dat de game objecten worden weergegeven in de volgorde waarin ze worden aangemaakt_ -> mist het woord "in"; dat in de game...

Reply:
Het gaat niet over objecten in de game, maar game objecten.